h2o-algos/src/main/java/hex/generic/GenericModelOutput.java

```diff
@@ -73,27 +77,67 @@ private ModelMetrics determineModelmetricsType(final MojoModelMetrics mojoMetric
                 auc._auc = binomial._auc;
                 auc._pr_auc = binomial._pr_auc;
                 auc._gini = binomial._gini;
-                return new ModelMetricsBinomialGeneric(null, null, mojoMetrics._nobs, mojoMetrics._MSE,
-                        _domains[_domains.length - 1], Double.NaN, // TODO: sigma
-                        auc, binomial._logloss, convertTable(binomial._gains_lift_table),
-                        new CustomMetric(mojoMetrics._custom_metric_name, mojoMetrics._custom_metric_value), binomial._mean_per_class_error,
-                        convertTable(binomial._thresholds_and_metric_scores), convertTable(binomial._max_criteria_and_metric_scores),
-                        convertTable(binomial._confusion_matrix));
+                if (mojoMetrics instanceof MojoModelMetricsBinomialGLM) {
+                    assert modelAttributes instanceof GLMModelAttributes;
+                    final GLMModelAttributes glmModelAttributes = (GLMModelAttributes) modelAttributes;
+                    final MojoModelMetricsBinomialGLM glmBinomial = (MojoModelMetricsBinomialGLM) binomial;
+                    return new ModelMetricsBinomialGLMGeneric(null, null, mojoMetrics._nobs, mojoMetrics._MSE,
+                            _domains[_domains.length - 1], glmBinomial._sigma,
+                            auc, binomial._logloss, convertTable(binomial._gains_lift_table),
```

Comment:
Pavel:  Is pr_auc also a binomial metric?

Reply:
It is and we do have it. It's present in the `MojoModelMetricsBinomial` and it is filled few lines above this line:

```
                auc._auc = binomial._auc;
                auc._pr_auc = binomial._pr_auc;
                auc._gini = binomial._gini;
```

because this part is common for both GLM binomial and "normal" binomial.